src/common/HaskellDo/Toolbar/State.hs

```diff
@@ -30,8 +33,14 @@ initialState = State
     , projectConfig = ""
     , projectOpened = False
     , createProject = False
+    , directoryExists = False
+    , directoryList = ([], [])
     }
 
+
+lastProjectFile :: FilePath
+lastProjectFile = "lastproject"
```

Comment:
@NickSeagull also here, do you think we should have a file containing these constants? we have this one repeated in two files